Components/input.css

```diff
@@ -22,7 +22,9 @@
   padding: 0.5rem;
   border: 1px solid var(--black);
```

Comment:
you can change the border of the input field to something light. Currently, it's blending with the background.

Reply:
Done!